packages/components/datetime/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@contentful/f36-datetime",
+  "version": "4.0.0",
+  "private": true,
+  "description": "Forma 36: DateTime component",
+  "scripts": {
+    "build": "parcel build index.ts"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.6.2",
+    "@contentful/f36-core": "^4.0.0",
+    "@contentful/f36-tokens": "^4.0.0",
+    "emotion": "^10.0.17"
+  },
+  "peerDependencies": {
+    "dayjs": "^1.10.6",
```

Comment:
I think it should be in `dependencies`

Reply:
Oh really? Wouldn't that duplicate dayjs if the project using this already have dayjs? I'm always confused with this stuff

But I can fix it of course

Reply:
I changed it, but let me know about the duplication thing

Reply:
It won't duplicate. `^` version is rather relaxed range